MCprep_addon/util_operators.py

```diff
@@ -225,6 +225,15 @@ def execute(self, context):
 			return {'CANCELLED'}
 		return {'FINISHED'}
 
+class MCPREP_OT_local_refresh(bpy.types.Operator):
+	bl_idname = "mcprep.refresh_data"
+	bl_label = "Locally refresh MCprep data"
```

Comment:
Thinking about it further, maybe this would be more intuitive of a name? Mostly because it clearly articulates we're really pulling data from somewhere and not just some in-memory isolated reloading, and that we are actually pulling resources not just data (resources will also resonate more with users, being used to resource packs, thus self-describing a bit better). You could make it a bit more concise by calling it `mcprep.extract_mc_resources`.

```suggestion
class MCPREP_OT_extract_minecraft_resources(bpy.types.Operator):
	bl_idname = "mcprep.extract_minecraft_resources"
	bl_label = "Extract resources from Minecraft"
```